cmd/internal/api/versions.go

```diff
@@ -547,11 +547,14 @@ func GetRequestOptions(r *http.Request) (v2.RequestOptions, error) {
 	}
 	count := r.URL.Query().Get("count")
 	if len(count) != 0 {
-		n, err := strconv.ParseUint(count, 10, 32)
+		n, err := strconv.Atoi(count)
 		if err != nil {
 			return opt, fmt.Errorf("failed to parse 'count' option: %v", count)
```

Comment:
Worth wrapping the error with %w?

Reply:
Happy to do more work for this later, but this PR was an attempt to be surgical and just get the use-case working. I don't know that this HTTP API has a good set of tests as it is ...